src/components/global/NavBar/NavLInks.vue

```diff
@@ -3,14 +3,15 @@ import { defineComponent } from 'vue';
 
 export default defineComponent({
   name: 'NavLinks',
-  data() {
-    return {
-      urls: [
-        { id: 1, displayName: 'Movies', routeName: 'Movies' },
-        { id: 2, displayName: 'Screenings', routeName: 'Screenings' },
-        { id: 3, displayName: 'Contact us', routeName: 'Contact' }
-      ]
-    };
+  urls: [
+    { id: 1, displayName: 'Movies', routeName: 'Movies' },
+    { id: 2, displayName: 'Screenings', routeName: 'Screenings' },
+    { id: 3, displayName: 'Contact us', routeName: 'Contact' }
+  ],
+  computed: {
```

Comment:
no need to make it computed :) you can now get `urls` in the template just with `$options.urls` 👌